CONTRIBUTING.md

```diff
@@ -1,5 +1,61 @@
 # Contributing to rticonnext-examples
 
+## How to request a feature or enhacement
+
+If there is a new feature or enhacement that you would like to be added to
+the examples, feel free to open an issue in this repository and we will take
+a look of your issue as soon as possible. Please, do not forget to use a
+title and clear description about your feature or enhacement you would like
+us to include.
+
+## How to submit a change
+
+If you have any change to submit do not hesitate in opening a Pull Request.
+Before you submit the Pull Request, please be sure you are using a correct
+code style/format.
+
+-   If you submit changes into a `Python` script or you are adding a new
+one, be aware we will check the style/format with an automatic workflow. If you
+are not sure you are using a correct style/format, install
+[black](https://pypi.org/project/black/) and execute it in your local machine
+to ensure the correctness of your code.
+
+-   If you submit changes into a `C/C++` source file, you also have to use a
+correct style/format of the code. For this purpose, we are using
+[clang-format](https://clang.llvm.org/docs/ClangFormat.html). The rules of
+`clan-format` are listed into the `.clang-format` file.
+
+-   If you submit changes into a `Markdown` file, we are using some format rules
+listed into `.markdownlint.json`. The tool we are using for checking the
+`Markdown` rules is
+[markdown-cli](https://www.npmjs.com/package/markdownlint-cli).
+
+To execute the linters, we use a script located in
+`resources/ci_cd/linux_format.py` in the automatic workflow. You can
+also execute this script in your local machine.
+
+```plaintext
+Usage: python resources/ci_cd/linux_format.py [options]
+
+Options:
+ -r     <commit hash range>  Commits hash range on which the linters
```

Comment:
how do we specify a range? <commit hash1>:<commit hash2> ? or <commit hash1>...<commit hash2>. Please, add how do we specify this range

Reply:
@ManuelJNunez you can delete this usage and just explain that they can learn how to use it running the "--help" command.